fleet-be/src/main/java/com/everis/training/fleet/business/fleet/control/CustomerRepository.java

```diff
@@ -40,14 +39,14 @@ public void deleteVehicle(Integer id) {
 
     public void reserveCar(Reserve reserve) {
         Customer c;
-        c=em.find(Customer.class, reserve.getIdcustomer());
+        c=em.find(Customer.class, reserve.getIdCustomer());
         c.setVehicle(reserve.getVin());
         em.merge(c);
```

Comment:
Performing "em.merge" here is redundant since the entity customer is already attached to the JPA session. Please, remove it.